metrics/prometheus/prometheus.go

```diff
@@ -137,7 +148,7 @@ const (
 )
 
 // NewMetrics initializes a new Prometheus metrics instance with preloaded label values.
-func NewMetrics(cfg config.PrometheusMetrics, disabledMetrics config.DisabledMetrics, syncerKeys []string) *Metrics {
+func NewMetrics(cfg config.PrometheusMetrics, disabledMetrics config.DisabledMetrics, syncerKeys []string, moduleStageNames map[string][]string) *Metrics {
```

Comment:
I don't see any account-related module metrics being captured for prometheus like I see for influx (`go_metrics`). Did you intentionally leave that out because of cardinality or was it just an oversight?

Reply:
As I understood, PBS-Go intentionally does not collect account adapter metrics for prometheus because of the cardinality. As we have a similar situation with modules I decided to follow this convention. Furthermore, as I understood from a previous statement from Scott, using accounts as labels in prometheus is not an option.